doc/Makefile

```diff
@@ -1,177 +1,25 @@
-# Makefile for Sphinx documentation
```

Comment:
This makefile was being ignored as `cylc make-docs` invoked the builder directly.